proposals/nnnn-remote-control-seat.md

```diff
@@ -0,0 +1,233 @@
+# Remote Control - Seat
+
+* Proposal: [SDL-NNNN](nnnn-remote-control-seat.md)
+* Author: [Zhimin Yang](https://github.com/yang1070)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / iOS / Android / RPC ]
+
+## Introduction
+
+SDL remote control allows a user to control certain modules of the vehicle. This proposal allows mobile applications control powered seats within a vehicle (while the vehicle is in parking). A mobile application can send a remote control command to adjust seat positions, turn on or off cooled/heated seat, or change massage settings. The application can also get the status of each seat as well as receive notifications once the status of any seat changes.
```

Comment:
Please update "This proposal allows mobile applications control powered seats..." to "This proposal allows mobile applications to control powered seats..."

Reply:
updated, thank you